Sources/Models/BlogPost.swift

```diff
@@ -45,3 +46,67 @@ public struct BlogPost: Equatable, Identifiable {
     case stephen
   }
 }
+
+extension BlogPost: TestDependencyKey {
+  public static var testValue = {
+    [
```

Comment:
I'm using the `BlogPost` type as a place to express the `blogPosts` dependency, but if that feels weird I can just define a little private enum key.